test/extended/operators/olm.go

```diff
@@ -71,3 +71,57 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 		})
 	}
 })
+
+// This context will cover test case: OCP-23440, author: jiazha@redhat.com
+var _ = g.Describe("[Feature:Platform] the end user use OLM", func() {
+	defer g.GinkgoRecover()
+
+	var oc = exutil.NewCLI("olm-23440", exutil.KubeConfigPath())
+
+	var (
+		buildPruningBaseDir = exutil.FixturePath("testdata", "olm")
+		operatorGroup       = filepath.Join(buildPruningBaseDir, "operatorgroup.yaml")
+		catalogSourceConfig = filepath.Join(buildPruningBaseDir, "csc.yaml")
```

Comment:
Should we be testing CSCs? The type is not provided by OLM and is not required to test if we can install an operator from an existing CatalogSource.

Reply:
Yes, you're right. We don't need to create CSC for the test in 4.2, but it's a necessary for 4.1. Remove it, thanks!